what4/what4.cabal

```diff
@@ -1,6 +1,6 @@
 Cabal-version: 2.4
 Name:          what4
-Version:       1.3.0.0.99
+Version:       1.4
 Author:        Galois Inc.
 Maintainer:    jhendrix@galois.com, rdockins@galois.com
 Copyright:     (c) Galois, Inc 2014-2022
```

Comment:
Let's bump the copyright and add you and I (or anyone else you can think of) as Maintainer.

Reply:
I have added us as maintainers, and I have also updated the copyright years to 2023.